src/map/Map.js

```diff
@@ -1,3 +1,24 @@
+import {
+	bind, extend, setOptions, stamp, indexOf,
+	requestAnimFrame, cancelAnimFrame
+} from '../core/Util';
+
+import {Evented} from '../core/Events';
+import {EPSG3857} from '../geo/crs/CRS.EPSG3857';
+import {Point, toPoint} from '../geometry/Point';
+import {Bounds, toBounds} from '../geometry/Bounds';
+import {LatLng, toLatLng} from '../geo/LatLng';
+import {LatLngBounds, toLatLngBounds} from '../geo/LatLngBounds';
+
+import {
+	any3d as isAny3D,

```

Comment:
Yeah, I'd like to avoid `as` too, but the problem is that without namespaces, some of the exported names become awkward, and I wanted each file to export names that will end up in the namespace. See https://github.com/Leaflet/Leaflet/pull/4989/commits/a397b6385ed7eafa21da54471196657e448503c0 for how the `Browser` module looked, for example. Maybe we'll want to migrate to new names, but in a breaking way, in future.